README.md

```diff
@@ -1 +1,2 @@
-# Furniture-LP
\ No newline at end of file
+# LandingPage-Furniture
+A simple furniture landing page using HTML  and CSS
```

Comment:
README should be more Descriptive and carries more information about the project, The Dependencies, The Contributors, The Usage, including but not limited to Authors and acknowledgements. Please make changes to this file and commit. Use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make a better README Documentation